packages/tools/client-debugger/client-debugger-view/src/components/ContainerView.tsx

```diff
@@ -139,53 +140,26 @@ export interface PanelViewSelectionMenuProps {
  * @internal
  */
 export function PanelViewSelectionMenu(props: PanelViewSelectionMenuProps): React.ReactElement {
-	const { currentSelection, updateSelection } = props;
-
-	const options: IOverflowSetItemProps[] = Object.entries(PanelView).map(([_, flag]) => ({
+	const { updateSelection } = props;
+	const [selectedValue, setSelectedValue] = React.useState<TabValue>("none");
```

Comment:
The props already pass in the current selection. We shouldn't need to introduce any new state here.